docs/core/versions/remove-runtime-sdk-versions.md

```diff
@@ -201,7 +201,7 @@ The parent directories for the SDK and runtime are listed in the output from the
 
 ## .NET Core Uninstall Tool
 
-The [.NET Core Uninstall Tool](../additional-tools/uninstall-tool.md) (`dotnet-core-uninstall`) lets you remove .NET Core SDKs and Runtimes from a system. A collection of options is available to specify which versions should be uninstalled.
+The [.NET Core Uninstall Tool](../additional-tools/uninstall-tool.md) (`dotnet-core-uninstall`) lets you remove .NET Core SDKs and runtimes from a system. A collection of options is available to specify which versions should be uninstalled.
```

Comment:
@thraka and I discussed that when Runtimes was the product, we'd capitalize it

Reply:
So this is sort of like saying "lets you remove Visual Studios from a system"? That doesn't seem quite right.

Reply:
Not really. You can download .NET Core SDKs or you can download .NET Core Runtimes:
e.g. https://dotnet.microsoft.com/download/dotnet-core/3.1